style.css

```diff
@@ -452,3 +450,89 @@ button:disabled {
   line-height: 43px;
   text-align: center;
 }
+
+#form-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 70px 6.4%;
+  width: 100%;
+  gap: 58px;
+}
+
+.form-heading {
+  text-align: center;
+  max-width: 600px;
+  min-width: 327px;
+  max-height: 262px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: var(--blue);
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  min-width: 327px;
+  max-width: 600px;
+  height: 330px;
+  width: 100%;
+}
+
+input {
+  width: 100%;
+  max-width: 600px;
+  min-width: 327px;
+  height: 48px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  padding-left: 16px;
+  display: flex;
+  align-items: center;
+  color: var(--gray);
+  border: none;
+  border-bottom: 1px solid #dbd8d7;
+}
+
+textarea {
+  width: 100%;
+  min-width: 327px;
+  min-height: 114px;
+  font-family: 'Inter', sans-serif;
+  font-style: italic;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  padding-left: 16px;
+  color: var(--dark-grey);
+  background: #fbf8f7;
+  border: none;
+}
```

Comment:
- [ ] Excellent job 🥇🥇 Can you please fix the colors of the placeholders of all fields to get a design close to the Figma template as much as possible as mentioned in the project requirements ? 😄⛏ You can go through [this guide](https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder) to learn how to style the placeholder ℹ You can use something like 👇

```
input::placeholder {
  // given color  
}

textarea::placeholder {
  // given styles
}
```
Your design | Given design 
---| ---|
![image](https://github.com/tajemouti/portfolio/assets/106119105/b262f81a-1ee6-4a7b-9921-e180f3b719e5) | ![image](https://github.com/tajemouti/portfolio/assets/106119105/a1078c22-96fe-461c-a1e4-ec7a58baa7fa)